frontend/src/component/changeRequest/ChangeRequestOverview/ChangeRequestOverview.tsx

```diff
@@ -306,175 +338,244 @@ export const ChangeRequestOverview: FC = () => {
+                                    }
+                                    show={
+                                        <ConditionallyRender
+                                            condition={Boolean(scheduledAt)}
+                                            show={
+                                                <StyledButton
+                                                    variant='outlined'
+                                                    onClick={() =>
+                                                        setShowRejectScheduledDialog(
+                                                            true,
+                                                        )
+                                                    }
+                                                    disabled={disabled}
+                                                >
+                                                    Reject changes
+                                                </StyledButton>
+                                            }
+                                            elseShow={
+                                                <StyledButton
+                                                    variant='outlined'
+                                                    onClick={onCancel}
+                                                    disabled={disabled}
+                                                >
+                                                    Cancel changes
+                                                </StyledButton>
+                                            }
+                                        />
+                                    }
+                                />
+                            </StyledButtonBox>
+                        </StyledInnerContainer>
+                    </StyledPaper>
+                    <ConditionallyRender
+                        condition={
+                            changeRequest.state === 'Approved' &&
+                            approversEnabled
+                        }
+                        show={
+                            <StyledPaper sx={{ mt: 2 }} elevation={0}>
+                                <StyledInnerContainer sx={{ pt: 0, pb: 1.5 }}>
+                                    <StyledOuterContainer>
+                                        <StyledButtonContainer
+                                            bgColor={
+                                                theme.palette.primary.main!
+                                            }
+                                            svgColor={
+                                                theme.palette.background.paper
+                                            }
+                                        >
+                                            <ApprovedIcon
+                                                style={{
+                                                    transform: `scale(1.5)`,
```

Comment:
:thinking: this doesn't look like a common practice in the UI.

Reply:
It's taken directly from the other place the icon was in use, to look the same